skimage/measure/_regionprops.py

```diff
@@ -789,11 +844,14 @@ def regionprops_table(label_image, intensity_image=None,
             list(properties) + [prop.__name__ for prop in extra_properties]
         )
     if len(regions) == 0:
-        label_image = np.zeros((3,) * label_image.ndim, dtype=int)
-        label_image[(1,) * label_image.ndim] = 1
+        ndim = label_image.ndim
+        label_image = np.zeros((3,) * ndim, dtype=int)
+        label_image[(1,) * ndim] = 1
         if intensity_image is not None:
-            intensity_image = np.zeros(label_image.shape,
-                                       dtype=intensity_image.dtype)
+            intensity_image = np.zeros(
+                    label_image.shape + intensity_image.shape[ndim:],
+                    dtype=intensity_image.dtype
+                )
```

Comment:
```suggestion
                label_image.shape + intensity_image.shape[ndim:],
                dtype=intensity_image.dtype
            )
```

Reply:
This one was *not* PEP8 compliant! Fixing...

Reply:
... But I'm leaving the double indent, I actually like it much better than black's preference of single indent. Both are PEP8 compliant.